modules/standard/List.chpl

```diff
@@ -36,9 +36,9 @@
 
       - insert
       - remove
-      - sort
       - pop
       - clear
+      - sort (*unstable*)
```

Comment:
I worry that calling out 'unstable' in the docs like this is going to be confusing, since stable/unstable is already overloaded for sorts (beyond Chapel)